math/wxMaxima/Portfile

```diff
@@ -1,112 +1,78 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
 PortSystem          1.0
 
 PortGroup           active_variants 1.1
 PortGroup           wxWidgets 1.0
 PortGroup           github 1.0
 
 name                wxMaxima
-version             18.02.0
+version             20.04.0
 maintainers         nomaintainer
 license             GPL-2
-# openssl is a dependency of curl in cmake
-license_noconflict  openssl
+license_noconflict  openssl # openssl is a dependency of curl in cmake, so build only
 platforms           darwin
 categories          math aqua
 description         Graphical user interface for Maxima based on wxWidgets
-homepage            http://andrejv.github.io/wxmaxima/
-
-subport wxMaxima-devel {
-    conflicts       wxMaxima
-    long_description ${description}: \
-                    Provides the development version, which is typically updated every day.
-
-    # git describe --tags : Version-16.12.x-1834-gb32fed0
-    github.setup    andrejv wxmaxima b32fed0d535c05d589d1f573d2c0f4f839180d18
-    checksums       rmd160  86fcf85ddae42e84884a708d55acff393152ed23 \
-                    sha256  e34256294c51a37d80947b92d3cd979456bf0c8ae53a21ce6cbab63d99143bf2 \
-                    size    12739940
-    version         18.02-dev-20180616
-    revision        1
-
-    patchfiles      patch-devel-src_Dirstructure.cpp.diff
-    post-patch {
-        reinplace -W ${worksrcpath} "s|@@PREFIX@@|${prefix}|g" \
-                                    src/Dirstructure.cpp
-    }
-
-    livecheck.type  none
-}
+homepage            https://github.com/wxMaxima-developers/wxmaxima
```

Comment:
This `homepage` is set to this value by the `github.setup` and so this line should be removed.

Reply:
Sorry, a victim of the cmake port group trial. Done now.